main.js

```diff
@@ -46,22 +56,27 @@ const newBook = () => {
     };
     bookc.addBook(bookObj);
   }
-};
+});
 
 // ***********
 
+window.remove = (id) => {
+  bookc.removeBook(id);
+};
+
 // Locale storage
 
 window.addEventListener('DOMContentLoaded', () => {
+  recordSec.style.display = 'block';
+  addForm.style.display = 'none';
+  contactSec.style.display = 'none';
   const books = JSON.parse(localStorage.getItem('data'));
```

Comment:
- [ ] This line raises an error because of the localStorage is empty at app startup and then using `forEach` on this variable will not work because it is not yet an array.
<img width="452" alt="Screenshot 2022-12-10 at 12 46 55 AM" src="https://user-images.githubusercontent.com/40503704/206813440-c7740a37-c32f-45ca-bf33-ef79a47ec023.png">